### PR TITLE
feat(1493): Display the job name and description in the UI

### PR DESCRIPTION
**Pull request by yakanechi**
## Context
Implement the following feature.
https://github.com/screwdriver-cd/screwdriver/issues/1493

## Objective
Add a description item in the job tooltip in the UI.

Short description.
スクリーンショット 2023-07-10 13 33 47

Long description (round 25 chars)
スクリーンショット 2023-07-10 13 33 59

Long description (Displayed the full description with a click)
スクリーンショット 2023-07-10 13 34 07

## References
https://github.com/screwdriver-cd/screwdriver/issues/1493

## License
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/workflow-tooltip/component.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ export default Component.extend({
   showTooltip: false,
   showToggleModal: false,
   left: equal('showTooltipPosition', 'left'),
+  is: false,
+  // isDescriptionClicked: false,
```
Comment (y-oksaku) [inline app/components/workflow-tooltip/component.js:12]: Are these used?

File: app/components/workflow-tooltip/styles.scss [hunk 2]
```diff
@@ -51,6 +51,14 @@
       border-width: 10px;
       margin-left: -10px;
     }
+
+    .nowrap {
+      white-space: nowrap;
+    }
```
Comment (y-oksaku) [inline app/components/workflow-tooltip/styles.scss:57]: Maybe, you can use `text-nowrap` bootstrap class.

File: app/components/workflow-tooltip/component.js (hunk 1 above)
Comment (yakanechi) [reply app/components/workflow-tooltip/component.js]: No.
I have deleted there.

File: app/components/workflow-tooltip/styles.scss (hunk 2 above)
Comment (yakanechi) [reply app/components/workflow-tooltip/styles.scss]: I accepted your advice.

**Review by y-oksaku — APPROVED** [review]

File: app/components/workflow-tooltip/component.js [hunk 3]
```diff
@@ -26,12 +28,36 @@ export default Component.extend({
...
     const isDisabled = get(this, 'tooltipData.job.isDisabled');
 
     return isDisabled ? 'ENABLED' : 'DISABLED';
   }),
+  jobDescription: computed(
+    'tooltipData.job.description',
+    'viewDescriptionMaxLength',
+    function jobDescription() {
+      let description = String(get(this, 'tooltipData.job.description'));
+      // maxLength is the umber of characters.
```
Comment (tkyi) [inline app/components/workflow-tooltip/component.js:49]:
```suggestion
      // maxLength is the number of characters.
```

**Review by tkyi — APPROVED** [review]
small comment

**Review by kumada626 — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**